app/components/utility/flash_message_component.rb

```diff
@@ -38,7 +38,7 @@ def render?
 private
 
   def messages
-    flash[message_key]
+    flash[message_key].html_safe
```

Comment:
Fine as long as we don't display user input. The flash messages can also be an array. That will raise an error when calling `.html_safe`. You might want to check if it's not an array or call it in the `header`/`body` method.